tests/workflows/basic_code_execution_node/workflow_with_code.py

```diff
@@ -0,0 +1,18 @@
+from vellum.workflows import BaseWorkflow
+from vellum.workflows.nodes.displayable.code_execution_node import CodeExecutionNode
+from vellum.workflows.state.base import BaseState
+
+base_module = __name__.split(".")[:-1]
+
+
+class SimpleCodeExecutionNode(CodeExecutionNode[BaseState, int]):
+    code = 'def main() -> str:\n    return "Hello, World!"\n'
```

Comment:
[oos] I think I really want to start whitelisting a few attributes for block strings over normal strings, this being one of the first